### ISSUE TITLE
Future Development: Is the Project Still Ongoing?

### ISSUE THREAD

**Issue by GeradeHouse** [facebookresearch/demucs#554]
## ❓ Questions

(Hi @adefossez and team,

First off, I wanted to express my gratitude for all the hard work you and the contributors (shoutout to @keunwoochoi, @famzah, and others) have put into Demucs. It's been incredible to see the progress and updates over the years.

I've been closely following the release notes and noticed the latest update was on 22/02/2023. I was wondering if there are any upcoming versions or enhancements in the pipeline for Demucs?

I genuinely hope the project continues to grow and evolve.)

**Comment by adefossez** [comment]
Thanks for your kind words! I am not actively working on this, and I do not have plans for the coming year sadly.

**Comment by Dyslexicon** [comment]
@adefossez maybe next iteration could be a generative rather than masking model.
I would love to see Demucs leapfrog this year's models like MVSEP and SAMI-Bytedance, in version 5.0
I said in another thread, all those Rockband, Guitar Hero etc videogame stems could be employed for extra training data.
Theres also those pay-for-access stem repositories that could be utilized as training data.
Very enthusiastic for any further development of Demucs. 
Hope you decide to keep pushing forward!

**Comment by CarlGao4** [comment]
@Dyslexicon can you find some high quality music stem datasets? I'd like to train some new models that can separate other stems, but the only "high quality" model I can find is musdb ...

**Comment by Dyslexicon** [comment]
There was a huge repository of videogame stems on Fretsonfire.com but theyre gone now, maybe someone has an archive on mega or google drive. Just do a search I know theres some big archives online that charge a small fee for access. 

I'd love to see all this added to Demucs training data in the next iteration. 
If team involvement is needed I can help with mixing stems for each song back down to full-mixdown files for the algorithm to compare to the individual stems during training. 20 years audio editing experience not a n00b hehe

Demucs 5.0 generative model trained on 2TB of data lol

**Comment by CarlGao4** [comment]
@Dyslexicon Maybe we can have deeper discussion via email? gaoweiqicarl1@163.com

**Comment by mdlaat** [comment]
> I said in another thread, all those Rockband, Guitar Hero etc videogame stems could be employed for extra training data.

Stems of those are not in lossless quality. Using it for training will result in unwanted artifacts in the results, see p.e. https://github.com/ZFTurbo/MVSEP-MDX23-music-separation-model/issues/2

**Comment by Dyslexicon** [comment]
Yes the upcoming *Diffusion* model shall be trained only on Lossless
